matlab/CARFAC_Test.m

```diff
@@ -313,6 +329,74 @@
 return
 
 
+function status = test_IHC3(do_plots)
+% Test: Make sure that IHC (inner hair cell) runs as expected.
+% Two-cap version with receptor potential; slightly different blips.
+
+status = 0;
+
+test_freqs = [300, 3000];
+for k = 1:length(test_freqs)
+  switch test_freqs(k)
+    case 300
+      expected_results = [ ...
+        [1.898725, 450.438353];
```

Comment:
These goldens look quite different to the 2 cap - which i guess is expected but I don't know how to "know" they're good/bad.

Reply:
I'm not thinking the numbers are good yet.  They are just what makes it pass for now.  And when I take out that fs = 48000 they're probably wrong anyway.  So treat these as placeholders.